tensorflow/examples/label_image/main.cc

```diff
@@ -103,7 +103,10 @@ Status ReadTensorFromImageFile(const string& file_name, const int input_height,
     image_reader = DecodePng(root.WithOpName("png_reader"), file_reader,
                              DecodePng::Channels(wanted_channels));
   } else if (tensorflow::StringPiece(file_name).ends_with(".gif")) {
-    image_reader = DecodeGif(root.WithOpName("gif_reader"), file_reader);
+    // gif decoder returns 4-D tensor, remove the first dim
+    image_reader = Squeeze(root.WithOpName("squeeze_first_dim"),
```

Comment:
@freedomtan Perhaps you can add a line to the documentation at the top to mention that animated GIFs are not expected/supported.